lib/wallaby/browser.ex

```diff
@@ -1043,7 +1102,7 @@ defmodule Wallaby.Browser do
     end
   end
 
-  defp execute_query(parent, query) do
+  def execute_query(parent, query) do
```

Comment:
as they are implemented as macros we'd need this to be public, which begs the question if it's really the right interface to use here (I just used what `has?` uses), if this should be public facing, or if we should make another version/wrapper that is public facing.